resiliency/README.md

```diff
@@ -0,0 +1,7 @@
+# Dapr Resiliency
+
+In this Quickstart, you will observe Dapr resiliency capabilities by simulating a system failure. You have the option of using the service invocation quickstarts to demonstrate Dapr resiliency between service-to-service communication, or the state management quickstart to demonstrate resiliency between apps and components. 
+
+### Select Resiliency Quickstart
+- For service-to-service resiliency, see [here](./service-to-service-resiliency.md)
```

Comment:
For the Doc (not github) - I hope we can "flatten" all this into one walkthrough.

Reply:
Should really take this into account. https://github.com/dapr/cli/issues/953 The Components path has become a hack.

Reply:
Agreed, we should call it resources-path or specs-path